app/models.py

```diff
@@ -124,6 +124,7 @@ def __str__(self) -> str:
 class Request(models.Model):
     """ Request model """
 
+    request_id = models.AutoField(primary_key=True, editable=False)
```

Comment:
Why the change?

Reply:
proper naming, re: pylint standards. primary_key as a name didn't exist as the models id name so it wasn't working at all. Decided to just give append the models name and id as the primary_key names.